### ISSUE TITLE
Getting 52.01 on VQA-CP

### ISSUE THREAD

**Issue by LinkToPast1990** [itaigat/removing-bias-in-multi-modal-classifiers#1]
Hi, @itaigat , thanks for your great work. I tried to reproduce 54.55 on VQA-CP but got 52.01, using the following command:
`python main.py --output saved_models --seed 0 --cache_features --eval_each_epoch --inf_lambda 4e-17
`
4e-17 is from https://papers.nips.cc/paper/2020/file/20d749bc05f47d2bd3026ce457dcfd8e-Supplemental.pdf

Did I miss any hyper-parameter? or anything wrong? 
Thanks again!

```
epoch 1, time: 313.99
	train_loss: inf, score: 26.21
	eval score: 40.08 (90.22)
epoch 2, time: 332.40
	train_loss: 3.70, score: 40.59
	eval score: 47.00 (90.22)
epoch 3, time: 332.09
	train_loss: 3.36, score: 43.40
	eval score: 49.11 (90.22)
epoch 4, time: 307.44
	train_loss: 3.16, score: 45.35
	eval score: 50.93 (90.22)
epoch 5, time: 347.27
	train_loss: 3.02, score: 47.94
	eval score: 51.68 (90.22)
epoch 6, time: 355.29
	train_loss: 2.91, score: 50.06
	eval score: 52.35 (90.22)
epoch 7, time: 395.44
	train_loss: 2.81, score: 51.89
	eval score: 53.25 (90.22)
epoch 8, time: 403.33
	train_loss: 2.73, score: 53.98
	eval score: 53.24 (90.22)
epoch 9, time: 399.54
	train_loss: 2.65, score: 55.95
	eval score: 52.60 (90.22)
epoch 10, time: 400.44
	train_loss: 2.58, score: 57.57
	eval score: 53.38 (90.22)
epoch 11, time: 398.15
	train_loss: 2.52, score: 59.31
	eval score: 53.18 (90.22)
epoch 12, time: 394.01
	train_loss: 2.46, score: 61.08
	eval score: 51.72 (90.22)
epoch 13, time: 388.91
	train_loss: 2.40, score: 62.33
	eval score: 51.76 (90.22)
epoch 14, time: 385.31
	train_loss: 2.35, score: 64.18
	eval score: 51.76 (90.22)
epoch 15, time: 382.76
	train_loss: 2.30, score: 65.47
	eval score: 52.01 (90.22)
```

**Comment by LinkToPast1990** [comment]
I also tried to reproduce Learned-Mixin +H: 52.013 but got 51.30, using
`python main.py --output saved_models --seed 0 --cache_features --eval_each_epoch --inf_lambda 0.0
`
And this time, the train_loss at epoch 1 is not inf.
```
epoch 1, time: 208.88
	train_loss: 5.97, score: 21.93
	eval score: 41.64 (90.22)
epoch 2, time: 202.84
	train_loss: 3.69, score: 39.99
	eval score: 47.91 (90.22)
epoch 3, time: 194.42
	train_loss: 3.34, score: 43.83
	eval score: 49.92 (90.22)
epoch 4, time: 196.24
	train_loss: 3.14, score: 45.88
	eval score: 51.77 (90.22)
epoch 5, time: 195.25
	train_loss: 3.00, score: 48.45
	eval score: 51.93 (90.22)
epoch 6, time: 194.18
	train_loss: 2.89, score: 50.85
	eval score: 52.90 (90.22)
epoch 7, time: 191.64
	train_loss: 2.79, score: 52.94
	eval score: 53.39 (90.22)
epoch 8, time: 190.26
	train_loss: 2.71, score: 54.91
	eval score: 53.63 (90.22)
epoch 9, time: 190.23
	train_loss: 2.63, score: 56.53
	eval score: 52.61 (90.22)
epoch 10, time: 191.60
	train_loss: 2.56, score: 58.69
	eval score: 53.43 (90.22)
epoch 11, time: 194.45
	train_loss: 2.50, score: 60.18
	eval score: 52.82 (90.22)
epoch 12, time: 196.09
	train_loss: 2.44, score: 61.72
	eval score: 51.42 (90.22)
epoch 13, time: 196.22
	train_loss: 2.38, score: 63.54
	eval score: 51.76 (90.22)
epoch 14, time: 198.41
	train_loss: 2.32, score: 65.26
	eval score: 51.80 (90.22)
epoch 15, time: 196.05
	train_loss: 2.28, score: 66.48
	eval score: 51.30 (90.22)
```

**Comment by itaigat** [comment]
Hi, 

It looks like with our regularization you get 53.38 while without you get 51.30. I believe this is related to random seed, we ran our experiments on RTX 2080.

**Comment by LinkToPast1990** [comment]
Without the regularization, i got 53.63 at epoch 8, if we do not take the result at epoch 15.

**Comment by LinkToPast1990** [comment]
So, could you kindly provide the pretrained model as you mentioned on readme.md? https://gofile.io/d/FbLhKD
